lighthouse-core/gather/gatherers/image-elements.js

```diff
@@ -99,6 +109,8 @@ function getCSSImages(allElements) {
       // CSS Images do not expose natural size, we'll determine the size later
       naturalWidth: 0,
       naturalHeight: 0,
+      attributeWidth: element.getAttribute('width') || '',
```

Comment:
I'm not sure we should even populate the width/height attribute for CSS images since it doesn't carry any weight, just always leave as `''`? we'll need to solve CSS a different way :)

Reply:
I agree! I'll switch it to that